lib/kontena/plugin/cloud/platform/wizard_command.rb

```diff
@@ -22,12 +24,19 @@ def execute
 
     self.name = prompt.ask("Name:") unless self.name
     self.organization = prompt_organization unless self.organization
-    self.region = prompt_region unless self.region
-    self.initial_size = prompt_initial_size unless self.initial_size
+    self.type = prompt_type unless self.type
+    self.region = prompt_region if self.region.nil? && self.type != 'mini'
+    unless self.initial_size
+      if self.type == 'mini'
+        self.initial_size = 1
+      else
+        self.initial_size = 3
+      end
+    end
 
     platform = nil
-    spinner "Creating platform master #{pastel.cyan(name)} to region #{pastel.cyan(region)}" do
-      platform = create_platform(name, organization, initial_size, region)
+    spinner "Creating platform master #{pastel.cyan(self.name)} to region #{pastel.cyan(self.region)}" do
```

Comment:
spinner returns the block result, so you can also do `platform = spinner "....."` instead of presetting with nil.